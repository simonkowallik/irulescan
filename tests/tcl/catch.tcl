catch {
    expr 1
} result

catch {
    expr 2
}
