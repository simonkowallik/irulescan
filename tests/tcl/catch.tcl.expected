WARNING: Unquoted expr at `1` in `expr 1`
WARNING: Unquoted expr at `2` in `expr 2`

